### ISSUE TITLE
`SCDNA` dataset to be added

### ISSUE THREAD

**Issue by kasra-keshavarz** [kasra-keshavarz/datatool#14]
# Problem

The `SCDNA` dataset to be added to the list of datasets.

# Extra information
The relevant information could be found in the following: https://uofs-comphyd.github.io/group_datasets/SCDNA

**Comment by kasra-keshavarz** [comment]
The structure of the dataset proved to be not super user-friendly. This needs to be dealt with separately.

**Comment by ShervanGharari** [comment]
Indeed, SCDNA is not a homogenous set of stations, a subset of stations can be extracted for different organizations and variables. This might be helpful in dealing with SCDNA:
https://github.com/ShervanGharari/EASYMORE/blob/main/examples/04_Thiessen_Polygons_SCDNA.ipynb
It is a light dataset, easier to take care of it outside of the datatool.
